.github/workflows/build.yml

```diff
@@ -15,14 +15,14 @@ on:
 jobs:
   build:
 
-    runs-on: macos-latest
+    runs-on: macos-13
 
     steps:
     - uses: actions/checkout@v2
 
     - uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: 14.1
+        xcode-version: 15.0.1
```

Comment:
15.1.0?

Reply:
I kept it on 15.0.1 because the slow performance didn't seem to affect this extensions tests passing. I'm open to swapping to 15.1 to match other extensions though? I mostly thought it would be nice to keep on parity with the release Xcode version if possible.

Reply:
I kept it on 15.0.1 because the slow performance didn't seem to affect this extensions tests passing. I'm open to swapping to 15.1 to match other extensions though? I mostly thought it would be nice to keep on parity with the release Xcode version if possible.